### PR TITLE
Add necessary comments for RNG class

### PR DESCRIPTION
**Pull request by fanshiqing**

### CODE REVIEW

File: dl/src/main/scala/com/intel/analytics/bigdl/utils/RandomGenerator.scala [hunk 1]
```diff
@@ -183,6 +183,9 @@ class RandomGenerator() {
     this
   }
 
+  /**
```
Comment (yiheng) [inline dl/src/main/scala/com/intel/analytics/bigdl/utils/RandomGenerator.scala]: It can't pass the format check. Use java-doc style comment

Comment (linbojin) [inline dl/src/main/scala/com/intel/analytics/bigdl/utils/RandomGenerator.scala]: Do you use scalafmt: https://github.com/olafurpg/scalafmt A good format tool.

Comment (yiheng) [inline dl/src/main/scala/com/intel/analytics/bigdl/utils/RandomGenerator.scala]: @linbojin We just check the style in the build. To do code format, I personally use IDE reformat function...

**Comment by yiheng** [comment]
Jenkins test pass. Merge the code.

File: dl/src/main/scala/com/intel/analytics/bigdl/utils/RandomGenerator.scala (hunk 1 above)
Comment (linbojin) [inline dl/src/main/scala/com/intel/analytics/bigdl/utils/RandomGenerator.scala]: Yes, same to me, scalafmt is already integrated inside latest intellij idea 😄

**Pull request merged**